### ISSUE TITLE
[Not Sure if this is an MI or Indium issue] MI+Indium Crashes the client with immersive portals.

### ISSUE THREAD

**Issue by TheArmaggedon** [AztechMC/Modern-Industrialization#368]
Here is the exception 
Reported exception thrown!
net.minecraft.class_148: Rendering item
	at net.minecraft.class_918.method_4026(class_918.java:360)
	at net.minecraft.class_918.method_36543(class_918.java:340)
	at net.minecraft.class_918.method_27951(class_918.java:336)
	at net.minecraft.class_329.method_1762(class_329.java:1274)
	at net.minecraft.class_329.method_1759(class_329.java:596)
	at net.minecraft.class_329.method_1753(class_329.java:221)
	at net.minecraft.class_757.method_3192(class_757.java:862)
	at net.minecraft.class_310.method_1523(class_310.java:1143)
	at net.minecraft.class_310.method_1514(class_310.java:734)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NullPointerException: Cannot invoke "net.fabricmc.fabric.api.renderer.v1.model.SpriteFinder.find(net.fabricmc.fabric.api.renderer.v1.mesh.QuadView, int)" because the return value of "link.infra.indium.other.SpriteFinderCache.forBlockAtlas()" is null
	at link.infra.indium.renderer.render.VertexConsumerQuadBufferer.bufferQuad(VertexConsumerQuadBufferer.java:65)
	at link.infra.indium.renderer.render.ItemRenderContext.bufferQuad(ItemRenderContext.java:179)
	at link.infra.indium.renderer.render.ItemRenderContext.renderQuad(ItemRenderContext.java:205)
	at link.infra.indium.renderer.render.ItemRenderContext.renderMeshQuad(ItemRenderContext.java:231)
	at link.infra.indium.renderer.render.ItemRenderContext$MeshConsumer.accept(ItemRenderContext.java:262)
	at link.infra.indium.renderer.render.ItemRenderContext$MeshConsumer.accept(ItemRenderContext.java:250)
	at aztech.modern_industrialization.pipes.impl.PipeMeshCache.draw(PipeMeshCache.java:196)
	at aztech.modern_industrialization.pipes.impl.PipeModel.emitItemQuads(PipeModel.java:132)
	at link.infra.indium.renderer.render.ItemRenderContext.renderModel(ItemRenderContext.java:111)
	at net.minecraft.class_918.handler$chk000$hook_renderItem(class_918.java:1557)
	at net.minecraft.class_918.method_23179(class_918.java:132)
	at net.minecraft.class_918.method_4021(class_918.java:307)
	at net.minecraft.class_918.method_4026(class_918.java:350)
	... 12 more

This is the full crash report

---- Minecraft Crash Report ----
// Shall we play a game?

Time: 7/24/22, 9:04 PM
Description: Rendering item

java.lang.NullPointerException: Cannot invoke "net.fabricmc.fabric.api.renderer.v1.model.SpriteFinder.find(net.fabricmc.fabric.api.renderer.v1.mesh.QuadView, int)" because the return value of "link.infra.indium.other.SpriteFinderCache.forBlockAtlas()" is null
	at link.infra.indium.renderer.render.VertexConsumerQuadBufferer.bufferQuad(VertexConsumerQuadBufferer.java:65)
	at link.infra.indium.renderer.render.ItemRenderContext.bufferQuad(ItemRenderContext.java:179)
	at link.infra.indium.renderer.render.ItemRenderContext.renderQuad(ItemRenderContext.java:205)
	at link.infra.indium.renderer.render.ItemRenderContext.renderMeshQuad(ItemRenderContext.java:231)
	at link.infra.indium.renderer.render.ItemRenderContext$MeshConsumer.accept(ItemRenderContext.java:262)
	at link.infra.indium.renderer.render.ItemRenderContext$MeshConsumer.accept(ItemRenderContext.java:250)
	at aztech.modern_industrialization.pipes.impl.PipeMeshCache.draw(PipeMeshCache.java:196)
	at aztech.modern_industrialization.pipes.impl.PipeModel.emitItemQuads(PipeModel.java:132)
	at link.infra.indium.renderer.render.ItemRenderContext.renderModel(ItemRenderContext.java:111)
	at net.minecraft.class_918.handler$chk000$hook_renderItem(class_918.java:1557)
	at net.minecraft.class_918.method_23179(class_918.java:132)
	at net.minecraft.class_918.method_4021(class_918.java:307)
	at net.minecraft.class_918.method_4026(class_918.java:350)
	at net.minecraft.class_918.method_36543(class_918.java:340)
	at net.minecraft.class_918.method_27951(class_918.java:336)
	at net.minecraft.class_329.method_1762(class_329.java:1274)
	at net.minecraft.class_329.method_1759(class_329.java:596)
	at net.minecraft.class_329.method_1753(class_329.java:221)
	at net.minecraft.class_757.method_3192(class_757.java:862)
	at net.minecraft.class_310.method_1523(class_310.java:1143)
	at net.minecraft.class_310.method_1514(class_310.java:734)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at link.infra.indium.renderer.render.VertexConsumerQuadBufferer.bufferQuad(VertexConsumerQuadBufferer.java:65)
	at link.infra.indium.renderer.render.ItemRenderContext.bufferQuad(ItemRenderContext.java:179)
	at link.infra.indium.renderer.render.ItemRenderContext.renderQuad(ItemRenderContext.java:205)
	at link.infra.indium.renderer.render.ItemRenderContext.renderMeshQuad(ItemRenderContext.java:231)
	at link.infra.indium.renderer.render.ItemRenderContext$MeshConsumer.accept(ItemRenderContext.java:262)
	at link.infra.indium.renderer.render.ItemRenderContext$MeshConsumer.accept(ItemRenderContext.java:250)
	at aztech.modern_industrialization.pipes.impl.PipeMeshCache.draw(PipeMeshCache.java:196)
	at aztech.modern_industrialization.pipes.impl.PipeModel.emitItemQuads(PipeModel.java:132)
	at link.infra.indium.renderer.render.ItemRenderContext.renderModel(ItemRenderContext.java:111)
	at net.minecraft.class_918.handler$chk000$hook_renderItem(class_918.java:1557)
	at net.minecraft.class_918.method_23179(class_918.java:132)
	at net.minecraft.class_918.method_4021(class_918.java:307)
	at net.minecraft.class_918.method_4026(class_918.java:350)
	at net.minecraft.class_918.method_36543(class_918.java:340)
	at net.minecraft.class_918.method_27951(class_918.java:336)
	at net.minecraft.class_329.method_1762(class_329.java:1274)
	at net.minecraft.class_329.method_1759(class_329.java:596)
	at net.minecraft.class_329.method_1753(class_329.java:221)

-- Item being rendered --
Details:
	Item Type: item_pipe
	Item Damage: 0
	Item NBT: null
	Item Foil: false
Stacktrace:
	at net.minecraft.class_918.method_4026(class_918.java:350)
	at net.minecraft.class_918.method_36543(class_918.java:340)
	at net.minecraft.class_918.method_27951(class_918.java:336)
	at net.minecraft.class_329.method_1762(class_329.java:1274)
	at net.minecraft.class_329.method_1759(class_329.java:596)
	at net.minecraft.class_329.method_1753(class_329.java:221)
	at net.minecraft.class_757.method_3192(class_757.java:862)
	at net.minecraft.class_310.method_1523(class_310.java:1143)
	at net.minecraft.class_310.method_1514(class_310.java:734)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected level --
Details:
	All players: 1 total; [class_746['Snawk'/502, l='ClientWorld minecraft:overworld', x=-399.33, y=43.00, z=-160.46]]
	Chunk stats: Client Chunks (ImmPtl) 5
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,89,0), Section: (at 0,9,0 in 0,5,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 23023 game time, 23023 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:453)
	at net.minecraft.class_310.method_1587(class_310.java:2288)
	at net.minecraft.class_310.method_1514(class_310.java:753)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 2
	Reload reason: manual
	Finished: No
	Packs: Default, Fabric Mods, 2e00901230c87c4f4e30e6dabc4e87e70f2c874d

-- System Details --
Details:
	Minecraft Version: 1.19
	Minecraft Version ID: 1.19
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 994296800 bytes (948 MiB) / 3087007744 bytes (2944 MiB) up to 5368709120 bytes (5120 MiB)
	CPUs: 8
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 3750H with Radeon Vega Mobile Gfx  
	Identifier: AuthenticAMD Family 23 Model 24 Stepping 1
	Microarchitecture: unknown
	Frequency (GHz): 2.30
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: AMD Radeon(TM) RX Vega 10 Graphics
	Graphics card #0 vendor: Advanced Micro Devices, Inc. (0x1002)
	Graphics card #0 VRAM (MB): 512.00
	Graphics card #0 deviceId: 0x15d8
	Graphics card #0 versionInfo: DriverVersion=27.20.22025.2003
	Graphics card #1 name: NVIDIA GeForce GTX 1660 Ti
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 4095.00
	Graphics card #1 deviceId: 0x2191
	Graphics card #1 versionInfo: DriverVersion=31.0.15.1659
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.40
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.40
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 19775.65
	Virtual memory used (MB): 17050.16
	Swap memory total (MB): 3968.00
	Swap memory used (MB): 419.10
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx5G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		actuallyunbreaking: Actually Unbreaking 1.19-0.1.0
		adorn: Adorn 3.5.4+1.19
		adventurez: AdventureZ 1.4.14
		ae2: Applied Energistics 2 12.0.1-alpha
		architectury: Architectury 5.8.29
		badpackets: Bad Packets 0.1.2
		bclib: BCLib 2.0.16
		betteranimalsplus: Better Animals Plus 1.19-11.0.7
		betterend: Better End 2.0.9
		betternether: Better Nether 7.0.6
		bigbeacons: Big Beacons 1.2.6
		biomemakeover: Biome Makeover 1.19-1.5.8
		bosses_of_mass_destruction: Bosses of Mass Destruction (Beta) 1.4.1-1.19
			maelstrom_library: Maelstrom Library 1.3-1.19-pre1
			multipart_entities: MultipartEntities 1.2-1.19-pre1
		byg: Oh The Biomes You'll Go 2.0.0.5
		cardinal-components: Cardinal Components API 5.0.1
			cardinal-components-base: Cardinal Components API (base) 5.0.1
			cardinal-components-block: Cardinal Components API (blocks) 5.0.1
			cardinal-components-chunk: Cardinal Components API (chunks) 5.0.1
			cardinal-components-entity: Cardinal Components API (entities) 5.0.1
			cardinal-components-item: Cardinal Components API (items) 5.0.1
			cardinal-components-level: Cardinal Components API (world saves) 5.0.1
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 5.0.1
			cardinal-components-world: Cardinal Components API (worlds) 5.0.1
		cloth-config: Cloth Config v6 7.0.72
			cloth-basic-math: cloth-basic-math 0.6.1
		craftpresence: CraftPresence 1.8.8
		dashloader: DashLoader 4.0.0+1.19
			com_github_luben_zstd-jni: zstd-jni 1.5.2-2
			dev_quantumfusion_hyphen: Hyphen 0.2.2
			dev_quantumfusion_taski: Taski 1.1.0
		dungeons_arise: When Dungeons Arise 2.1.51
		dynamicfps: Dynamic FPS 2.2.0
			com_moandjiezana_toml_toml4j: toml4j 0.7.2
		entityculling: EntityCulling-Fabric 1.5.2-mc1.19
			com_logisticscraft_occlusionculling: occlusionculling 0.0.6-SNAPSHOT
		fabric: Fabric API 0.56.1+1.19
			fabric-api-base: Fabric API Base 0.4.9+e62f51a3a9
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.7+9ff28f40a9
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.14+b2a4a624a9
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.18+9ff28f40a9
			fabric-command-api-v1: Fabric Command API (v1) 1.2.6+f71b366fa9
			fabric-command-api-v2: Fabric Command API (v2) 2.1.2+0d55f585a9
			fabric-commands-v0: Fabric Commands (v0) 0.2.23+df3654b3a9
			fabric-containers-v0: Fabric Containers (v0) 0.1.27+df3654b3a9
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.1.1+82552130a9
			fabric-convention-tags-v1: Fabric Convention Tags 1.0.8+37622d24a9
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.3+bd0a0d4aa9
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.1.0+a680b9b4a9
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.27+72da3b3da9
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.16+9ff28f40a9
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.26+9ff28f40a9
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.25+df3654b3a9
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.19+18990361a9
			fabric-item-api-v1: Fabric Item API (v1) 1.5.4+767db26ea9
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.24+9ff28f40a9
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.18+9ff28f40a9
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.16+df3654b3a9
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.1.0+33fbc738a9
			fabric-loot-api-v2: Fabric Loot API (v2) 1.0.2+9e7660c6a9
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.2+9e7660c6a9
			fabric-message-api-v1: Fabric Message API (v1) 1.0.1+513f4a59a9
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.9+33fbc738a9
			fabric-models-v0: Fabric Models (v0) 0.3.15+9ff28f40a9
			fabric-networking-api-v1: Fabric Networking API (v1) 1.0.27+7fe97409a9
			fabric-networking-v0: Fabric Networking (v0) 0.3.14+df3654b3a9
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.0.6+9ff28f40a9
			fabric-particles-v1: Fabric Particles (v1) 1.0.8+dc39553aa9
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.16+92cf9a3ea9
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.8+9ff28f40a9
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.7+c57dbccaa9
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.18+df3654b3a9
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.12+9ff28f40a9
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.5+9ff28f40a9
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.20+df3654b3a9
			fabric-rendering-v1: Fabric Rendering (v1) 1.10.14+9ff28f40a9
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.9+e62f51a3a9
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.2.6+9ff28f40a9
			fabric-textures-v0: Fabric Textures (v0) 1.0.18+9ff28f40a9
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.0.9+e62f51a3a9
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.1.1+9e7660c6a9
		fabric-language-kotlin: Fabric Language Kotlin 1.8.1+kotlin.1.7.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.3
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.3
		fabricloader: Fabric Loader 0.14.8
		ferritecore: FerriteCore 4.2.1
		geckolib3: Geckolib 3.1.5
			com_eliotlash_mclib_mclib: mclib 18
			com_eliotlash_molang_molang: molang 18
		immersive_portals: Immersive Portals 2.0.5
			imm_ptl_core: Immersive Portals Core 2.0.5
			q_misc_util: QMiscUtil 2.0.5
		incendium: Incendium + Seed Fix 5.1.0
		indium: Indium 1.0.6+mc1.19
		iris: Iris 1.2.5
			org_anarres_jcpp: jcpp 1.4.14
		java: OpenJDK 64-Bit Server VM 17
		krypton: Krypton 0.2.0
			com_velocitypowered_velocity-native: velocity-native 3.1.2-SNAPSHOT
		ksyxis: Ksyxis 1.1
		lazydfu: LazyDFU 0.1.3
		lithium: Lithium 0.8.1
		megane: megane 8.2.0
			megane-api: megane-api 8.2.0
			megane-applied-energistics-2: megane-applied-energistics-2 8.2.0
			megane-fabric-transfer: megane-fabric-transfer 8.2.0
			megane-reborn-core: megane-reborn-core 8.2.0
			megane-reborn-energy: megane-reborn-energy 8.2.0
			megane-runtime: megane-runtime 8.2.0
			megane-tech-reborn: megane-tech-reborn 8.2.0
			megane-vanilla: megane-vanilla 8.2.0
		minecraft: Minecraft 1.19
		modern_industrialization: Modern Industrialization 1.3.1
			magna: Magna 1.8.0-1.19
				omega-config: OmegaConfig 1.0.8
			noindium: No Indium? 1.1.0+1.19
			playerabilitylib: Pal 1.6.0
			team_reborn_energy: Energy 2.2.0
		modmenu: Mod Menu 4.0.4
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.5.5+a6d2f785a9
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.24+f05effafa9
		morevillagers: More Villagers 4.0.2
		mythicmounts: Mythic Mounts 1.19-7.0
		patchouli: Patchouli 1.19-74-FABRIC
			fiber: fiber 0.23.0-2
		phosphor: Phosphor 0.8.1
		quarryplus: QuarryPlus 19.1.610
		roughlyenoughitems: Roughly Enough Items 9.1.503
		roughlyenoughresources: Roughly Enough Resources 2.6.0
		seasons: Fabric Seasons 1.4-BETA+1.19
		simply_houses: Simply Houses 1.1.2-1.19
		sodium: Sodium 0.4.2+build.16
			org_joml_joml: joml 1.10.4
		techreborn: Tech Reborn 5.3.2
			reborncore: Reborn Core 5.3.2
		terrablender: TerraBlender 2.0.0.119
			com_electronwill_night-config_core: core 3.6.4
			com_electronwill_night-config_toml: toml 3.6.4
		toolleveling: ToolLeveling+ 1.19-1.3.3
		travelersbackpack: Traveler's Backpack 1.19-8.0.1
		trinkets: Trinkets 3.4.0
		voidz: VoidZ 1.0.8
		wolveswitharmor: Wolves With Armor 1.8.0-1.19
			maybe-data: Maybe data 1.3.2-1.19-rc2
			static-content: Static Content 1.0.1-1.16.2
				staticdata: Static Data 1.1.2
		wthit: wthit 5.6.1
		xaerominimap: Xaero's Minimap 22.12.0
		xaeroworldmap: Xaero's World Map 1.26.1
	Loaded Shaderpack: ComplementaryShaders_v4.5.1.zip
		Profile: MEDIUM (+1 option changed by user)
	Launched Version: fabric-loader-0.14.8-1.19
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2 GL version 3.2.0 NVIDIA 516.59, NVIDIA Corporation
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: English (US)
	CPU: 8x AMD Ryzen 7 3750H with Radeon Vega Mobile Gfx

**Comment by rdrpenguin04** [comment]
What were you doing when this happened?

**Comment by TheArmaggedon** [comment]
I was trying to join my private server. The server runs fine, it only crashes me (the client).

**Comment by TheArmaggedon** [comment]
Removing Indium and Sodium fixes the problem for me. However, I can no longer use Iris for the complementary shaders.

**Comment by Technici4n** [comment]
This is not an MI issue, see also PepperCode1/Continuity#120 as it is the same underlying issue. I am closing this as it is not an MI issue, but rather a Fabric API issue. **However, if anyone is able to consistently reproduce this issue, please leave a comment so that we can try to debug it and fix it in Fabric API.**

**Comment by itsbanditodorito** [comment]
hi! i think I was able to reproduce this issue? I'm not very adept at reading crash reports, nor logs, but I'm pretty sure that there is a problem regarding the spritefinder.

Like a few users in comments on [PepperCoed1/Continuity#120](https://github.com/PepperCode1/Continuity/issues/120), the problem did not occur with dashloader disabled. Though, unlike them, I do not use continuity. Also I do not use MI like in this thread. I am able to consistently trigger a crash with [this](https://pastebin.com/imN6XNJJ) crash report by attempting to access page 8 of my creative tab as long as dashloader is enabled in the modlist. Without dashloader, I am unable to reproduce this issue at all (so far).  I can provide full logs, rather than just the crash report, me attempting to reproduce the issue with and without dashloader, if needs be.

again I'm noob, and all I know for certain is that I searched this line to get here,

> java.lang.NullPointerException: Cannot invoke "net.fabricmc.fabric.api.renderer.v1.model.SpriteFinder.find(net.fabricmc.fabric.api.renderer.v1.mesh.QuadView, int)" because the return value of "link.infra.indium.other.SpriteFinderCache.forBlockAtlas()" is null

and that all references that I could find had dashloader in the modlist. 
Hope this helps, and very sorry if this is a completely unrelated issue.
